api/python/provisioner/commands/deploy_dual.py

```diff
@@ -189,10 +189,18 @@ def _run_states(self, states_group: str, run_args: run_args_type):
                     self._apply_state(
                         f"components.{state}", secondaries, stages
                     )
+                elif state in (
+                    "ha.cortx-ha",
```

Comment:
that code won't be run since the same condition is presented above, please check line 167

Reply:
comment is addressed